client/src/views/home/HomePage.vue

```diff
@@ -172,6 +172,14 @@ async function handleQuery(): Promise<void> {
     } else {
       console.error('Could not find the corresponding device');
     }
+  } else if (query.organizationId) {
```

Comment:
We should probably rename this to `saasOrganizationId` and check if the device belongs to a Saas organization. It is possible to get multiple devices belonging to an org with the same name on different servers.

Reply:
done